jwst/saturation/tests/test_saturation.py

```diff
@@ -42,6 +42,70 @@ def test_basic_saturation_flagging(setup_nrc_cube):
     assert np.all(output.groupdq[0, satindex:, 5, 5] == dqflags.group['SATURATED'])
 
 
+def test_low_saturation_flagging(setup_nrc_cube):
+    """Check that the saturation flag is set when a pixel value is zero or
+    negative."""
+
+    # Create inputs, data, and saturation maps
+    ngroups = 5
+    nrows = 2048
+    ncols = 2048
+    satvalue = 60000
+
+    data, satmap = setup_nrc_cube(ngroups, nrows, ncols)
+
+    # Add ramp values up to the saturation limit
+    data.data[0, 0, 5, 5] = 0  # Signal at bottom rail - low saturation
+    data.data[0, 1, 5, 5] = 0  # Signal at bottom rail - low saturation
+    data.data[0, 2, 5, 5] = 20
+    data.data[0, 3, 5, 5] = 40
+    data.data[0, 4, 5, 5] = 60
+
+    # frames that should be flagged as saturation (low)
+    satindxs = [0, 1]
+
+    # Set saturation value in the saturation model
+    satmap.data[500, 500] = satvalue
```

Comment:
If you reduce nrows/ncols above, then of course change this to something like `satmap.data[8, 8]`

Reply:
Done.